features-json/push-api.json

```diff
@@ -227,7 +227,7 @@
       "35":"n"
     },
     "and_chr":{
-      "49":"a #1"
+      "50":"y"
```

Comment:
Erm, rather, only data for what caniuse considers the "current" Android Chrome version is allowed. You'd need @Fyrd to bump that version from 49 to 50.